frontend/src/entities/configuration/dynamic-configuration.tsx

```diff
@@ -148,7 +150,57 @@ export function DynamicConfiguration () {
                     {<pre>TODO</pre>}
                 </ExpandableSection>
                 <ExpandableSection headerText='System Banner' variant='default' defaultExpanded>
-                    {<pre>TODO</pre>}
+                    <SpaceBetween direction='vertical' size='l'>
+                        <Toggle
+                            onChange={({ detail }) => {
+                                setFields({ 'configuration.SystemBanner.isEnabled': detail.checked });
+                            }}
+                            checked={state.form.configuration.SystemBanner!.isEnabled!}
+                        >
+                            Enable System Banner
+                        </Toggle>
+                        <FormField
+                            label='Banner Text'
+                        >
+                            <Input
+                                onChange={({ detail }) => {
+                                    setFields({ 'configuration.SystemBanner.text': detail.value });
+                                }}
+                                onBlur={() => touchFields(['configuration.SystemBanner.text'])}
+                                value={state.form.configuration.SystemBanner!.text}
+                                placeholder='Enter system banner text'
+                                disabled={!state.form.configuration.SystemBanner!.isEnabled}
```

Comment:
Remove `!` for SystemBanner if unnecessary for the whole file:

`SystemBanner!` -> `SystemBanner`